pkg/antctl/raw/serviceexport/command.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceexport
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8smcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var (
+	Command *cobra.Command
+	option  = &struct {
+		outputFormat string
+	}{}
+)
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "serviceexport",
+		Aliases: []string{"serviceexports"},
+		Short:   "Print Multi-cluster ServiceExports",
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 0 {
+				_ = runE(cmd, args[0])
+			} else {
+				_ = runE(cmd, "")
+			}
+		},
+	}
+	Command.Flags().StringVarP(&option.outputFormat, "output", "o", "", "Output format. Supported formats: json|yaml")
+}
+
+func runE(cmd *cobra.Command, ClusterID string) error {
+	configs, err := raw.ResolveMultiKubeconfigs(cmd)
+	if err != nil {
+		return err
+	}
+
+	var seviceExportMap map[string][]k8smcsv1alpha1.ServiceExport
+	seviceExportMap = make(map[string][]k8smcsv1alpha1.ServiceExport)
+	var seviceExportTotal []k8smcsv1alpha1.ServiceExport
+
+	for _, kubeconfig := range configs {
+
+		k8sClient, err := raw.AddSchemeToClientCfg(kubeconfig)
+		if err != nil {
+			return err
+		}
+
+		clusterClaimList := &multiclusterv1alpha1.ClusterClaimList{}
+
+		err = k8sClient.List(context.Background(), clusterClaimList, &client.ListOptions{Namespace: metav1.NamespaceAll})
+
+		clusterID := clusterClaimList.Items[0].Value
+
+		res := &k8smcsv1alpha1.ServiceExportList{}
+
+		err = k8sClient.List(context.TODO(), res, &client.ListOptions{Namespace: metav1.NamespaceAll})
+
+		if err != nil {
+			return err
+		}
+
+		results := res.Items
+		if ClusterID != "" {
+			if clusterID == ClusterID {
+				resExports := results
```

Comment:
What's your purpose to add a new var? I suppose you can use `results` directly.
btw, you are handling ServiceExport,not ResourceExport, you can name it as svcExports if needed.